Sources/SwiftSyntax/SyntaxNodes.swift.gyb

```diff
@@ -161,7 +161,7 @@ public struct ${node.name}: ${base_type}, _SyntaxBase, Hashable {
 %       if child.is_optional:
     let raw = newChild?.raw
 %       else:
-    let raw = newChild?.raw ?? ${make_missing_swift_child(child)}
```

Comment:
${make_missing_swift_child(child)} will be changed by this PR: https://github.com/apple/swift/pull/26982